pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -241,3 +258,88 @@ def cmd(self):
 
         cmd = (" ").join(cmd)
         return cmd
+
+
+class A11yCrawler(BokChoyTestSuite):
```

Comment:
Any reason why you wouldn't call this Pa11yCrawler? I was thinking for consistency's sake...